synapse/rest/client/pusher.py

```diff
@@ -132,48 +133,21 @@ async def on_POST(self, request: SynapseRequest) -> Tuple[int, JsonDict]:
         return 200, {}
 
 
-class PushersRemoveRestServlet(RestServlet):
+class LegacyPushersRemoveRestServlet(UnsubscribeResource, RestServlet):
```

Comment:
Ah, I see `UnsubscribeResource` isn't a `RestServlet`...that's annoying. I suspect this is fine though! 👍